labs/fabric-toolkit.md

```diff
@@ -0,0 +1,30 @@
+# Lab Name
+
+Hyperledger Fabric distributed deploy toolkit.
+
+# Short Description
+
+* This project is aimed to automate the deployment process of the hyperledger fabric.
+* The deployment tool provides basic mechanisms for deployment, repairment, and scaling of applications.
+
+# Scope of Lab
+
+This project is aimed to automate the deployment process of the hyperledger fabric. Hyperledger Fabric is a platform for distributed ledger solutions and the examples released can easily deploy the Hyperledger Fabric network in a standalone machine environment, including order, peer chaincode and so on.
+
+However, If you want to build up the Hyperledger Fabric network in the distributing development environments, it is not a easy task and you have to deploy the modules in each of the machine manually, which includes lots of configurations and commands and is inconvenient. Our preject is a tool to help you to install the hyperledger fabric in the distributing development environments easily and conveniently.
+
+The deployment tool provides basic mechanisms for deployment, repairment, and scaling of applications. Firstly, once you have completed the configuration of the orderer and peer，witch include the information of the role,the domain and ip address， the tool can deploy the orderer and the peer according the configuration you written by itself. Secondly, when the hyperledger fabric network is running， the tool can scale your hyperledger fabric network up and down with a simple command.
+
+# Initial Committers
+
+- [Ddpool0](https://github.com/Ddpool0) - Zheng Li
+- [rivulet2015](https://github.com/rivulet2015) - Yuejiang He
+- [zeoio](https://github.com/zeoio)
+
+# Sponsors
+
+- Credit Card Center of China Guangfa Bank (kinsleer@outlook.com)
+
+# Pre-existing Repository
+
+- https://github.com/zeoio/fabric-samples/tree/release-1.4/distributed-deploy
```

Comment:
This is the repository that will be transferred. This appears to be a sub-directory under a repository. If we were to transfer, we would need to have a separate repository with only the source code that would make up the lab. We should not be pulling the rest of fabric-samples into the lab.

Reply:
also, I see [this commit](https://github.com/hyperledger/fabric-samples/commit/83dbfcb2dcaaecbd3540d50a258e623483850686) does not have DCO sign-off. This would also need to be fixed.

Reply:
> also, I see [this commit](https://github.com/hyperledger/fabric-samples/commit/83dbfcb2dcaaecbd3540d50a258e623483850686) does not have DCO sign-off. This would also need to be fixed.

fixed.